packer/openstack.pkr.hcl

```diff
@@ -178,7 +178,7 @@ source "openstack" "openhpc" {
   ssh_bastion_private_key_file = var.ssh_bastion_private_key_file
   
   # Output image:
-  image_disk_format = "qcow2"
+  # image_disk_format = "qcow2"
```

Comment:
This should be set to null when use_blockstorage_volume == false, and otherwise got to "qcow2".